README.md

```diff
@@ -129,7 +168,7 @@ SQL database. If you need an example, you can use `people` and `wishes` models.
 Your answer...
 
 ```
-Replace this with your answer
+A schema is a collection of tables or objects in a database, which are related to each other. For example, you could have a table of people and each person would have a unique id, which would then be associated with several wishes by the column wisher_id on the wishes table.
```

Comment:
:+1: yes but, to be clear, a schema defines what a particular table will look like (i.e., what columns it will have).